hw03_altyyevaa.py

```diff
@@ -0,0 +1,224 @@
+# Author: Aziza Altyyeva
+# Username: altyyevaa
+#
+# Assignment: to create a creative project using functions, turtles, loops and etc.
+# Purpose: to practice what I have learned until today
+# Google Doc Link : https://docs.google.com/document/d/1qbbG98mVSgm59Zz0Qic8VSSfr2YC18v-gFslQo-V63c/edit
+
+
+import turtle
+
```

Comment:
Pay attention to the gray squigglies, they indicate style guidelines. In this case, they want two blank lines before all function definitions.

Reply:
Even though I fixed my indentations, added docstring to functions gray squiggles are still appearing in my code. As you said I left 2blank lines.

Reply:
Hold your mouse over the function name, and it'll tell you what style guideline is incorrect: 

<img width="451" alt="Screenshot 2024-02-05 at 10 17 30 PM" src="https://github.com/Berea-College-CSC-226/hw03-main/assets/1906220/61a762eb-de81-48ef-83e5-1927783d81a8">

Reply:
I fixed them, also are the green squigglies fine? Because when I created a turtle named lomi, it is showing Typo (Typo: In word 'lomi') . How it can be typo if i can name my turtle anything I want?

Reply:
It's a typo in that the term lomi isn't in the dictionary. But, you can ignore some squigglies, when you don't agree with them. In this case, I would ignore it because you want it called lomi, despite it not being a term in their dictionary.